module/Frontpage/src/Frontpage/Controller/OrganController.php

```diff
@@ -32,10 +32,16 @@ public function fraternityListAction()
 
     public function organAction()
     {
+        $type = $this->params()->fromRoute('type');
         $abbr = $this->params()->fromRoute('abbr');
         $organService = $this->getOrganService();
         try {
             $organ = $organService->findOrganByAbbr($abbr);
```

Comment:
Wouldn't it be better to adjust this function to include the type in the query?

Reply:
I wanted to keep the query as general as possible, but I think I've found a suitable solution in the next commit.